values/decoding/bit_stream_int_iterator.go

```diff
@@ -0,0 +1,73 @@
+package decoding
+
+import (
+	"fmt"
+	"io"
+
+	xio "github.com/xichen2020/eventdb/x/io"
+
+	bitstream "github.com/dgryski/go-bitstream"
+)
+
+// bitStreamIntIterator iterates through a bit encoded stream of ints.
+type bitStreamIntIterator struct {
+	bitReader           *bitstream.BitReader
+	bitsPerEncodedValue int
+	numEncodedValues    int
+
+	curr  int
+	count int
+	err   error
+}
+
+func newBitStreamIntIterator(
+	reader xio.Reader,
+	bitsPerEncodedValue int, // Number of bits per encoded value
```

Comment:
the variable names are pretty self-explanatory, comments don't seem necessary.

Reply:
Sure can remove.